app/views/shared/proxy_rules/_form.html.slim

```diff
@@ -1,7 +1,8 @@
 = form.inputs do
   = form.input :http_method, as: :select, collection: ProxyRule::ALLOWED_HTTP_METHODS, include_blank: false, label: 'Verb'
   = form.input :pattern
-  = form.input :metric_id, as: :select, collection: service.metrics, include_blank: false, label: 'Metric or Method to increment'
+  = form.input :redirect_url, as: :string if using_proxy_pro
```

Comment:
It should work with something like...
```suggestion
  = form.input :redirect_url, as: :string if owner.try(:using_proxy_pro?)
```
and we don't even need to set the local variable.